meson_options.txt

```diff
@@ -5,7 +5,11 @@ option('udevrulesdir', type : 'string', value : '',
        description: 'Where to install udev rules (if empty, query pkg-config(1))')
 
 option('utils', type : 'boolean', value : true,
-       description: 'Wheter or not to build and install helper programs')
+       description: 'Whether or not to build and install helper programs')
 
 option('examples', type : 'boolean', value : true,
-       description: 'Wheter or not to build example programs')
\ No newline at end of file
+       description: 'Whether or not to build example programs')
+
+option('no-root', type : 'boolean', value : false,
+       description: 'Install files without root permissions')
```

Comment:
Let's rename this to `useroot` (so you don't need an extra `useroot` variable in `meson.build`), defaut it to `True`, and describe it as *Set owner and setuid bits on installed files*.